src/AppBar/AppBar.js

```diff
@@ -39,12 +39,10 @@ export default class AppBar extends Component {
      * The CSS class name of the root element.
      */
     className: PropTypes.string,
-    primary: PropTypes.bool,
   };
 
   static defaultProps = {
     accent: false,
-    primary: true,
```

Comment:
When `accent` is set, `classes.primary` is disable, so the `primary` prop has no purpose (unless you think it should be possible to have no colors set?).